Libplanet/Assets/Currency.cs

```diff
@@ -24,12 +24,25 @@ namespace Libplanet.Assets
     /// Here is how US Dollar can be represented using <see cref="Currency"/>:
     /// <code>
     /// var USMint = new PrivateKey();
-    /// var USD = new Currency(ticker: "USD", decimalPlace: 2, minter: USMint.ToAddress());
+    /// var USD = Currency.Uncapped(ticker: "USD", decimalPlaces: 2, minter: USMint.ToAddress());
     /// var twentyThreeBucks = 23 * USD;
     /// // Or alternatively: USD * 23;
     /// // Or explicitly: new FungibleAssetValue(USD, 23, 0)
     /// </code>
     /// </example>
+    /// <remarks>There are two types of <see cref="Currency">Currencies</see>: capped and uncapped.
+    /// Capped currencies have a hard limit on the maximum minted amount, and uncapped currencies
+    /// do not have the said limit. To define a <see cref="Currency"/> you may call either of the
+    /// following.
+    /// <ul>
+    /// <li><see
+    /// cref="Capped(string,byte,ValueTuple{BigInteger, BigInteger},IImmutableSet{Address}?)"/>
+    /// </li>
+    /// <li><see cref="Capped(string,byte,ValueTuple{BigInteger, BigInteger},Address)"/></li>
+    /// <li><see cref="Uncapped(string,byte,IImmutableSet{Address}?)"/></li>
+    /// <li><see cref="Uncapped(string,byte,Address)"/></li>
+    /// </ul>
```

Comment:
I don't think this works? See other lists in XML docs. Also [this](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#list).

Reply:
Thanks for pointing out. Though, not that I prefer the usage of <ul/> and <li/> tags, but they actally does work on Rider (apparently XML docs are still XML, so regular tags do work.) Regardless <list> seems recommended so I'll use that.

Reply:
Seems like it doesn't work in VSCode. Also I was more concerned about generated documentation. 😗